### PR TITLE
RBD resize workflow

### PR DESCRIPTION
**Pull request by kamleshverma1**

### CODE REVIEW

**Comment by kamleshverma1** [comment]
@cloudbehl @gnehapk Please review it

File: src/modules/rbd/rbd-list/rbd-list.js [hunk 1]
```diff
@@ -18,6 +18,14 @@
             clusterObj;
 
         vm.createRbd = createRbd;
+        vm.onOpenRbdResizeModal = onOpenRbdResizeModal;
+        vm.resizeRBD = resizeRBD;
+        vm.viewTaskProgress = viewTaskProgress;
+        vm.resizeRBD = resizeRBD;
+        vm.resizeRbd = {"unit": "" , "size": 0};
+        vm.sizeUnits = ["bytes", "KB", "MB", "GB", "TB"];
```
Comment (gnehapk) [inline src/modules/rbd/rbd-list/rbd-list.js]: @kamleshverma1  As per my conversation with Ju, the size units should be "GB" and "TB". Does Ju has mentioned this change for size units.

File: src/modules/rbd/rbd-list/rbd-list.js [hunk 2]
```diff
@@ -82,6 +92,48 @@
...
             $state.go("create-rbd");
         }
+
+        function onOpenRbdResizeModal(rbdObject) {
+            vm.resizeRBDstep = 1;
+            vm.resizeRBDtaskSubmitted = false;
+            var size, sizeAndUnit = [], clusterObj;
+            vm.resizeRbd = rbdObject;
+            if(rbdObject.size) {
+                size = $filter('bytes')(rbdObject.size);
```
Comment (gnehapk) [inline src/modules/rbd/rbd-list/rbd-list.js]: @kamleshverma1 Please use consistent quotes across the views. Please replace ' with " wherever applicable.

File: src/modules/rbd/rbd-list/rbd-list.js [hunk 3]
```diff
@@ -82,6 +92,48 @@
...
+                vm.resizeRbd.clusterAvailable = clusterObj.utilization.available;
+            }
+        }
+
+
+        function resizeRBD() {
+            var sizeInBytes, sizeInMB, postData;
+            sizeInBytes = utils.convertToBytes(vm.resizeRbd.size, vm.resizeRbd.unit);
+            sizeInMB = (sizeInBytes / (1024*1024));
+            vm.resizeRbd.size = parseFloat(sizeInMB);
```
Comment (gnehapk) [inline src/modules/rbd/rbd-list/rbd-list.js]: @kamleshverma1 Do we need to post size in "MB" unit?

**Pull request merged**